script/metric/step_5_import_metric_value.rb

```diff
@@ -0,0 +1,108 @@
+
+require File.expand_path('../../../config/environment',  __FILE__)
+require 'colorize'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.